.gitignore

```diff
@@ -6,6 +6,6 @@ docs/_build
 .tox/
 *.pyc
 __pycache__
-
+.mypy_cache
```

Comment:
this gets created if you run mypy across the project (I do this sometimes even though we're not running it in CI)